versioned_docs/version-0.2.0/glossary.mdx

```diff
@@ -0,0 +1,11 @@
+## Glossary
+
+1. Slice Boundaries: The time required for processing depends upon the amount of data. When the amount of data
+to be replicated is massive, its efficient to seperate the data into slices and process it seperately.
+Slice boundaries are basically resources, with instructions that maintain the slices and also manages communication
+of the pods and services across clusters, edges, clouds, data centers etc.
+
+2. RFC1918 network addresses: The rapid growth of internet led to the exhaustion of the available ipV4 addresses, as a solution subnets 
```

Comment:
Can you rephrase it as `The rapid growth of Internet has led to the exhaustion ....`?